src/llmq/quorums.cpp

```diff
@@ -199,8 +199,7 @@ void CQuorumManager::TriggerQuorumDataRecoveryThreads(const CBlockIndex* pIndex)
     LogPrint(BCLog::LLMQ, "CQuorumManager::%s -- Process block %s\n", __func__, pIndex->GetBlockHash().ToString());
 
     for (auto& params : Params().GetConsensus().llmqs) {
-        // Process signingActiveQuorumCount + 1 quorums for all available llmqTypes
-        const auto vecQuorums = ScanQuorums(params.type, pIndex, params.signingActiveQuorumCount + 1);
+        const auto vecQuorums = ScanQuorums(params.type, pIndex, params.keepOldConnections);
```

Comment:
I though it was similar to previous cases where we keep mns for the recent _cycle_ to avoid signing failure but I'm no longer sure it's actually needed here... dropped the commit for now.